src/operators/_predicates.ts

```diff
@@ -244,6 +244,10 @@ export function $size(a: RawArray, b: number, options?: Options): boolean {
   return a.length === b;
 }
 
+function isOperatorAndNotObjectOperator(name: string): boolean {
```

Comment:
**recommendation**: I think this is better named `isNonBooleanOperator`

Reply:
This is a much better name! I made the change and also updated the comments and test name accordingly. Thanks!

Reply:
Thanks agains for this fix.